services/user.py

```diff
@@ -0,0 +1,49 @@
+from django.contrib.auth import get_user_model
+from typing import Optional
+
+
+User = get_user_model()
+
+
+def create_user(
+        username: str,
+        password: str,
+        email: Optional[str] = None,
+        first_name: Optional[str] = None,
+        last_name: Optional[str] = None
+) -> User:
+    user = User.objects.create_user(
+        username=username,
+        password=password,
+        email=email if email else "",
+        first_name=first_name if first_name else "",
+        last_name=last_name if last_name else ""
+    )
+    return user
+
+
+def get_user(user_id: int) -> User:
+    return User.objects.get(id=user_id)

```

Comment:
Using `get()` can raise a `DoesNotExist` exception if no `User` with the given `id` is found. Consider handling this exception to prevent runtime errors.